docs/reference/transform/examples.asciidoc

```diff
@@ -336,3 +337,116 @@ This {transform} makes it easier to answer questions such as:
 * Which client IPs have high error rates?
 
 * Which client IPs are interacting with a high number of destination countries?
+
+
+[[example-last-log]]
+== Finding the last log event from an IP address
+
+This example uses the web log sample dataset to find the last log from an IP 
+address. Let's use the `latest` type of {transform} which copies the most recent 
+documents from the source index to the destination index. Pick the `clientip` 
+field as the unique key; the data is grouped by this field. Select `@timestamp` 
+as the date field that sorts the data chronologically. The {transform} creates 
+the destination index that contains the latest login date for each client IP.
+
+[source,console]
+----------------------------------
+PUT _transform/last-log-from-clientip
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_logs"
+    ]
+  },
+  "latest": {
+    "unique_key": [ <1>
+      "clientip"
+    ],
+    "sort": "@timestamp" <2>
+  },
+  "frequency": "1m",
+  "dest": {
+    "index": "last-log-from-clientip"
+  },
+  "settings": {
+    "max_page_search_size": 500
+  }
```

Comment:
as said, we need `sync` in this example, running `latest` as batch is technically possible but I think most users want continuous.